recipes/cptec-sazonal/LICENSE.txt

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2015-2023, conda-forge
```

Comment:
this licence here does not seem to be correct, can you add a proper licence upstream please?
or at least fix this one?
because ``conda-forge`` is not the author, right? so we are not responsible for this package